widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx

```diff
@@ -138,6 +146,20 @@ export function WalletList(props: PropTypes) {
           }
         };
 
+        const blockchainDisplayName = getBlockchainDisplayNameFor(
+          experimentalChainWallet?.chain ?? '',
```

Comment:
I think passing empty string doesn't make sense. We could this method interface. If you are going to update this one, please also check similar cases.